specs/disk_inventory_etcd_structure.adoc

```diff
@@ -0,0 +1,138 @@
+= Disk inventory etcd structure
+
+This specification specifies the way in which disk inventory has to be structured
+in tendrl's central store and the details that has to be stored in the disk
+inventory
+
+== Problem description
+
+We should have a proper structure to our disk inventory in such a way that we
+store all the needed information in a easily consumable way.
+
+== Use Cases
+
+* This spec provides guidelines about how the disk information is to be
+stored in etcd.
+
+== Proposed change
+
+The disk inventory details will basically stored under 2 sections as follows:
+
+1. /nodes/<node-id>/disks/
+
+* In this location we store all the physical disks available on the node
+with a unique key which is a combination of manufacturer-id+vendor-id+
+device-serial-number this would be unique across tendrl. And this information
+should be available from hwinfo tool.
```

Comment:
We can use field from hwifo  
Device Files: /dev/sda, /dev/disk/by-id/ata-SAMSUNG_MZ7TE512HMHP-000L1_S1GJNSAG400778, /dev/disk/by-id/wwn-0x4d30445853885002

if you see here by-id is never change even disk is removed and connected again, problem is it gives two by-id
 In VM it give one by-id those format are different 
Device Files: /dev/vdh, /dev/disk/by-id/virtio-78065dbf-92db-4574-8, /dev/disk/by-path/virtio-pci-0000:00:0e.0

so i don't  know what to do